### ISSUE TITLE
firmware/scroll: What is the origin of `fuente6x8.png`?

### ISSUE THREAD

**Issue by ivan-tat** [zxdos/zxuno#67]
What is the origin of [`fuente6x8.png`](https://github.com/zxdos/zxuno/blob/master/firmware/scroll/fuente6x8.png)? I checked it with [`pngcheck`](http://freshmeat.net/projects/pngcheck) and this gave no results:
```
$ pngcheck -7qvt fuente6x8.png
File: fuente6x8.png (1300 bytes)
  chunk IHDR at offset 0x0000c, length 13
    96 x 40 image, 8-bit palette, non-interlaced
  chunk PLTE at offset 0x00025, length 768: 256 palette entries
  chunk bKGD at offset 0x00331, length 1
    index = 0
  chunk pHYs at offset 0x0033e, length 9: 2835x2835 pixels/meter (72 dpi)
  chunk tIME at offset 0x00353, length 7: 12 Jul 2016 21:59:19 UTC
  chunk IDAT at offset 0x00366, length 410
    zlib: deflated, 4K window, maximum compression
  chunk IEND at offset 0x0050c, length 0
No errors detected in fuente6x8.png (7 chunks, 66.1% compression).
```
As you can see - no Author / Copyright information found. Committers:
```
$ git log --pretty=format:"%as %an - %ae" fuente6x8.png|sort -u
2016-07-07 antoniovillena - espineter@yahoo.com
2016-07-09 antoniovillena - espineter@yahoo.com
2016-07-13 antoniovillena - espineter@yahoo.com
```

**Comment by antoniovillena** [comment]
This file was created by Quest. He is credited at the end of the scroll.tap

**Comment by ivan-tat** [comment]
Oh, thank you. I'll credit this person as an author of this file. Does he/she has a real name or is it a secret?

**Comment by ivan-tat** [comment]
I looked at `scroll` sources and found that information you mentioned is in the file [`string.asm`](https://github.com/zxdos/zxuno/blob/master/firmware/scroll/string.asm). I noticed a large list of contributors to ZX-Uno project there. I see the developers of the ZX-Uno were interested in crediting everyone who has contributed and helped this project to appear in real hardware. My respect for that! I also think it's important to mention every one, at least it's a pleasure to know that your work helped someone else. Unfortunately this list consists of many abstract nicknames without real names. Who knows - do they really exist? (I believe they do, but knowing only a nickname may be confusing - different people can have the same nickname in Internet and they can have different relation to this project and even hate it)

**Comment by ivan-tat** [comment]
Who knows - may be one day a game developer will see his/she work and will try to contact him/her for further cooperation.

**Comment by antoniovillena** [comment]
I explain. The list of contributors is a list of people who participate into the crowdfunding. We launched a crowdfunding of about 250 ZX-Unos through verkami:
https://www.verkami.com/projects/14074-zx-uno
So these guys are credited there, but they are not developers. It's an easteg egg (this list is hidden into the zx spectrum core) and we put the nick that they agreed. Note that some of them decided to put their complete names.

**Comment by antoniovillena** [comment]
The real name of Quest is "Samuel Baselga"

**Comment by ivan-tat** [comment]
Good. So, should I assume CC-BY-SA-4.0 International for all unsigned artwork?

**Comment by ivan-tat** [comment]
**...I'm still waiting for the answer, then I'll proceed further with automation of build process**.

**Comment by antoniovillena** [comment]
sorry. Yes, use CC-BY-SA-4.0 for all unsigned artwork

**Comment by ivan-tat** [comment]
Good.

**Comment by ivan-tat** [comment]
Closed by fea6c0f304d3dc6f361810993f1dcf399adc5b77
